client/Windows/wf_event.c

```diff
@@ -829,6 +831,11 @@ static BOOL wf_scale_mouse_pos(wfContext* wfc, UINT16* x, UINT16* y)
 	if (!settings)
 		return FALSE;
 
+	if (*(INT16*)x < 0)
```

Comment:
What is this for? `UINT16` can never be negative, so you effectively check for `x > INT16_MAX`